docs/updating/8-0.md

```diff
@@ -0,0 +1,182 @@
+
+```shell
+npm install vue@latest vue-router@latest
+```
+
+3. Update to the latest version of Ionic 8:
+
+```shell
+npm install @ionic/vue@next @ionic/vue-router@next
+```
+
+### Core
+
+1. Update to the latest version of Ionic 8:
+
+```shell
+npm install @ionic/core@next
+```
+
+## Recommended Changes
+
+The following changes are not required to update to Ionic 8 as your application will continue to work. However, we recommend making the following changes to ensure you can use the new features in Ionic 8.
+
+### Dark Theme
+
+In previous versions, it was recommended to define the dark theme in the following way:
+
+```css
+@media (prefers-color-scheme: dark) {
+  body {
+    /* global app variables */
+  }
+
+  .ios body {
+    /* global ios app variables */
+  }
+
+  .md body {
+    /* global md app variables */
+  }
+}
+```
+
+In Ionic Framework version 8, the dark theme is being distributed via css files that can be imported. Below is an example of importing a dark theme file in Angular:
+
+```css
+/* @import '@ionic/angular/css/themes/dark.always.css'; */
+/* @import "@ionic/angular/css/themes/dark.class.css"; */
+@import "@ionic/angular/css/themes/dark.system.css";
+```
+
+Notice that the dark theme is now applied to the `:root` selector instead of the `body` selector. The [`:root`](https://developer.mozilla.org/en-US/docs/Web/CSS/:root) selector represents the `<html>` element and is identical to the selector `html`, except that its specificity is higher.
+
+While migrating to include the new dark theme files is unlikely to cause breaking changes, these new selectors can lead to unexpected overrides if custom CSS variables are being set on the `body` element. We recommend updating any instances where global application variables are set to target the `:root` selector instead.
+
+For more information on the new dark theme files, refer to the [Dark Mode documentation](../theming/dark-mode).
+
+### Dynamic Font
+
+The `core.css` file has been updated to enable dynamic font scaling by default.
+
+The `--ion-default-dynamic-font` variable has been removed and replaced with `--ion-dynamic-font`.
+
+Developers who had previously chosen dynamic font scaling by activating it in their global stylesheets can revert to the default setting by removing their custom CSS. In doing so, their application will seamlessly continue utilizing dynamic font scaling as it did before. It's essential to note that altering the font-size of the html element should be avoided, as it may disrupt the proper functioning of dynamic font scaling.
+
+Developers who want to disable dynamic font scaling can set `--ion-dynamic-font: initial;` in their global stylesheets. However, this is not recommended because it may introduce accessibility challenges for users who depend on enlarged font sizes.
+
+For more information on the dynamic font, refer to the [Dynamic Font Scaling documentation](../layout/dynamic-font-scaling).
+
+## Updating Your Code
```

Comment:
This heading seems weird to me. Aren't the above examples also updating your code?

Reply:
I renamed it to "Required Changes" to make the distinction between the recommended changes and the required changes necessary for your app to compile. https://github.com/ionic-team/ionic-docs/pull/3475/commits/637b3b62e98061fa98fb301a34863c0da75e6ae3